Bottom: f57d9b1eb0e602f4ab6cf5348a0db7652efff815
Top:    f7e7da92789afb64547bee888535cd603ce791bc
Author: Robert Berger <Robert.Berger@ReliableEmbeddedSystems.com>
Date:   2024-01-05 11:05:39 +0000

poky-snapshot .66

Signed-off-by: Robert Berger <Robert.Berger@ReliableEmbeddedSystems.com>


---

diff --git a/meta-poky/conf/distro/poky.conf b/meta-poky/conf/distro/poky.conf
index 3b7bc66780..aa4c0e68c7 100644
--- a/meta-poky/conf/distro/poky.conf
+++ b/meta-poky/conf/distro/poky.conf
@@ -1,7 +1,7 @@
 DISTRO = "poky"
 DISTRO_NAME = "Poky (Yocto Project Reference Distro)"
-DISTRO_VERSION = "4.3+snapshot-${METADATA_REVISION}"
-DISTRO_CODENAME = "nanbield"
+DISTRO_VERSION = "4.3.66"
+DISTRO_CODENAME = "master-as-nanbield"
 SDK_VENDOR = "-pokysdk"
 SDK_VERSION = "${@d.getVar('DISTRO_VERSION').replace('snapshot-${METADATA_REVISION}', 'snapshot')}"
 SDK_VERSION[vardepvalue] = "${SDK_VERSION}"
